app/uk/gov/hmrc/helptosavestub/controllers/DESController.scala

```diff
@@ -16,25 +16,32 @@
 
 package uk.gov.hmrc.helptosavestub.controllers
 
-import cats.instances.list._
-import cats.instances.string._
-import cats.syntax.eq._
 import play.api.mvc.{Action, AnyContent, Request, Result}
+import uk.gov.hmrc.helptosavestub.config.AppConfig
 import uk.gov.hmrc.helptosavestub.util.Logging
 import uk.gov.hmrc.play.bootstrap.controller.BaseController
 
+import scala.collection.JavaConverters._
+
 // trait for controllers mimicking DES
-trait DESController { this: BaseController with Logging ⇒
+trait DESController {
+  this: BaseController with Logging with AppConfig ⇒
 
-  private val expectedAuthorisationHeader = List("Bearer test")
+  private val expectedDESHeaders: List[String] =
+    runModeConfiguration
+      .underlying
+      .getStringList("microservice.expectedDESHeaders")
+      .asScala
+      .toList
```

Comment:
you could also put this dependency in `build.sbt`:
```
  "com.github.kxbmap" %% "configs" % "0.4.4"
```
and do
```scala
  runModeConfiguration.underlying.get[List[String]]("microservice.expectedDESHeaders")
```
to get values from config as native scala types